src/middleware/injectCode.ts

```diff
@@ -113,12 +115,9 @@ export const injectCode = (root: string, baseURL: string, PHP: any, injectBodyOp
       createReadStream(filePath)
         .pipe(inject)
         .on('finish', () => {
-          if (!inject.injectTag) return next()
-          else {
-            res.type('html')
-            res.setHeader('Content-Length', inject.data.length)
-            res.send(inject.data)
-          }
+          res.type('html')
+          res.setHeader('Content-Length', inject.data.length)
+          res.send(inject.data)
```

Comment:
The conditional check here is removed because now the injection always happens, regardless if a closing tag was found or not.